index.html

```diff
@@ -3,10 +3,11 @@
 <head>
   <meta charset="utf-8">
   <link rel="stylesheet" href="styles.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"
```

Comment:
The syntax of the viewport is not correct at the end of the line. Please correct the syntax just like the example below:

<meta name="viewport" content="width=device-width, initial-scale=1.0">